src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -263,7 +263,7 @@ protected function buildFactoryReplacements()
             $replacements['{{ factoryImport }}'] = 'use Illuminate\Database\Eloquent\Factories\HasFactory;';
         } else {
             $replacements['{{ factoryCode }}'] = '//';
-            $replacements["\n{{ factoryImport }}"] = '';
+            $replacements["{{ factoryImport }}\n"] = '';
```

Comment:
This code does not pass the tests on the local Windows machine. (but in workflow, it passed).

![image](https://github.com/user-attachments/assets/c00b33a1-6319-487f-a557-3fe215b493f3)

On Windows `\r\n` is used as a new line and in UNIX `\n` is used as a new line that's why I used `PHP_EOL` to specify it and it was passing in the Windows machine.

But as you can see in the failing test in workflow, it is `\n` instead of `\r\n` so I doubt if it is running on Windows.

![image](https://github.com/user-attachments/assets/4f88944f-fd5a-476d-b3dd-22ed384bd82b)

Reply:
can you test againts `\r\n`, `\r` and `\n`

Reply:
see it passes with `\r\n`
![image](https://github.com/user-attachments/assets/71b1f018-2b08-4cab-8a04-3a48199d286f)

Reply:
Is it related to this #51958 PR